std_msgs/Header header
float64[3] principal_component
float64 eigenvalue